objects/Condor.py

```diff
@@ -0,0 +1,55 @@
+import os
+import fileinput
+import asyncio
+from .OPTSubprocess import OPTSubprocess
+
+class Condor(OPTSubprocess):
+    def __init__(self, id, logger, cwd, ip, port, poll_interval, workers, timeout):
+        super().__init__('Condor', id, logger)
+        self.ip = ip
+        self.port = port
+        self.poll_interval = poll_interval
+        self.workers = workers
+        self.timeout = timeout
+        self.cwd = cwd
+    
+    async def init_condor(self):
+        self.modify_timeout()
+        self.modify_submit()
+        await self.submit_condor()
+
+        
+    def modify_timeout(self):
+        printNext = False
```

Comment:
you might consider some of these functions/methods that are only called internally to be less public in scope by naming like `_modify_timeout()` etc. More stylistic probably than having meaningful performance impacts, but consider it